roles/network_plugin/cilium/templates/cilium-cr.yml.j2

```diff
@@ -63,6 +68,9 @@ rules:
 - apiGroups:
   - networking.k8s.io
   resources:
+{% if cilium_version | regex_replace('v') is version('1.7', '<') %}
+  - ingresses
```

Comment:
Are you sure about that ? I've check cilium template and could not find it.
And by the way, verbs get/list/watch are already added L102+ (and are indeed in 1.5x and 1.6x template)